src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/DecimalMathFuncs_NumberIsFloat.txt

```diff
@@ -250,9 +250,13 @@ Error({Kind:ErrorKind.Div0})
 >> Cot( [0] )
 Table({Value:Error({Kind:ErrorKind.Div0})})
 
+// .Net 4.6.2 returns -8.16588936419192E+15
+#DISABLE.NET: 462
```

Comment:
This is really not ideal. 
Especially since these .txts are shared by other backends (SQL, Jscript, and more to come) that also have numerical limitations. 

Can we handle this with a smart comparer? Ie, more tolerant to numerical diffs?

Reply:
Will be done in a future PR